src/pages/Cart/Cart.js

```diff
@@ -54,12 +67,23 @@ const CartWrapper = styled.div`
 const Content = styled.div`
   h3 {
     padding-bottom: 1.3rem;
-    border-bottom: 1px solid #e0e0e0;
+    border-bottom: 0.063px solid #e0e0e0;
     font-size: 1.2rem;
     font-weight: 700;
     margin-top: 5rem;
   }
 `;
+
 const ProductLists1 = styled(Content)``;
```

Comment:
```suggestion
const ProductLists1 = styled(Content)``;
```

추가적인 스타일을 보여줄 필요가 없다면 굳이 상속받아서 새로운 스타일드 컴포넌트를 만들 필요 없이 Content 컴포넌트를 그대로 사용해도 될 것 같네요!